test/translate_c.zig

```diff
@@ -217,6 +217,72 @@ pub fn addCases(cases: *tests.TranslateCContext) void {
         \\pub const OpenGLProcs = union_OpenGLProcs;
     });
 
+    cases.add_2("enums",
+        \\typedef enum {
+        \\    a,
+        \\    b,
+        \\    c,
+        \\} d;
+        \\enum {
+        \\    e,
+        \\    f = 4,
+        \\    g,
+        \\} h = e;
+        \\struct Baz {
+        \\    enum {
+        \\        i,
+        \\        j,
+        \\        k,
+        \\    } l;
+        \\    d m;
+        \\};
+        \\enum i {
+        \\    n,
+        \\    o,
+        \\    p,
+        \\};
+    , &[_][]const u8{
+        \\pub const a = enum_unnamed_1.a;
```

Comment:
These could be made to match C even bettee by using `@enumToInt`

Reply:
@andrewrk since integer operations on extern enums didn't (understandably) get accepted, could we do this instead?

Using enums as flags might be abuse of the type system but it is still a common thing. This would cut out all the `@enumToInt` conversions but you would still get the awkwardness abusing the type system brings with `@intToEnum`.

Reply:
I think it could make sense to translate the non-namespaced enum values as integers rather than the enum values (is that what you are proposing?). Then people have the choice of using enum literals (or fully qualified enum values) to get the enum type, or the non-namespaced values to get the integer.

The downside would be that using the integer types would not coerce to the enum type if, for example,  it was used as a function parameter type.

I'm definitely open to considering this but I don't think the choice is clear. Would you mind opening a separate proposal for this?